<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientTCPClass</class>
 <widget class="QMainWindow" name="ClientTCPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientTCP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#lnIPAddress[state = &quot;0&quot;]{
	background-color: rgb(255, 0, 0);

}
#lnIPAddress[state = &quot;1&quot;]{
	background-color: rgb(85, 255, 127);

}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IP Address:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lnIPAddress">
           <property name="inputMask">
            <string>000.000.000.000;_</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
           <property name="state" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spnPort">
           <property name="minimum">
            <number>1024</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btnConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpSendData">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Send Data</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Message</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lnMessage">
           <property name="inputMask">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
           <property name="state" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="btnSend">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="lstConsole"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ClientTCP.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked()</signal>
   <receiver>btnClear</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked(bool)</signal>
   <receiver>btnClear</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>pressed()</signal>
   <receiver>btnClear</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>released()</signal>
   <receiver>btnClear</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
